src/Aquifer.Common/Utilities/BibleWordIdentifier.cs

```diff
@@ -0,0 +1,182 @@
+﻿using Aquifer.Data.Enums;
+
+namespace Aquifer.Common.Utilities;
+
+/// <summary>
+/// Only use with Bible word identifiers, NOT Greek word identifers!!!
+/// The word identifier in the dbo.BibleVersionWords.WordIdentifier column has one more digit than the dbo.GreekNewTestamentWords.WordIdentifier column
+/// because Greek alignment data does not include word segments.
+/// Format for Bible word identifiers is BBCCCVVVWWWS.
+/// Format for Greek word identifiers is BBCCCVVVWWW and thus this class should not be used with Greek data.
+/// Key:
+///   B: Book
+///   C: Chapter
+///   V: Verse
+///   W: Word
+///   S: Word segment
+/// </summary>
+public sealed class BibleWordIdentifier
```

Comment:
I normally tend towards POCOs and separate utility classes but it made sense to me to contain this complexity in one class that is well tested and easy to use.